src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -312,7 +312,7 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
                         key => repository.ValueAsString(key).Value);
             }
 
-            project.BuildTargetFrameworksAndConfigurations(rawProject);
+            project.BuildTargetFrameworksAndConfigurations(rawProject, diagnostics);
```

Comment:
It doesn't look like we're doing anything with the `diagnostics` after this call. Should there be a check before we populate the dependencies below?